Mastodon/Scene/Profile/ProfileViewController.swift

```diff
@@ -971,11 +971,7 @@ extension ProfileViewController: DataSourceProvider {
         return nil
     }
     
-    func update(status: MastodonStatus) {
-        assertionFailure("Not required")
-    }
-    
-    func delete(status: MastodonStatus) {
+    func update(status: MastodonStatus, intent: MastodonStatus.UpdateIntent) {
```

Comment:
We might need to replace this with an actual implementation as posts don't get updated when I change them:

1. Go to my profile page.
1. Tap on one of my statuses (so that I see all its replies etc., you know)
1. Edit that very status
1. The status doesn't get updated on the Profile-screen

Reply:
Oops, will do 👍

Reply:
Fixed in https://github.com/mastodon/mastodon-ios/pull/1210/commits/4602eab547c12c558d1e6b30f1cb98d173a02791